lua/user/options.lua

```diff
@@ -1,46 +1,60 @@
+  cursorcolumn = true,                                 -- Enable a vertical column.
+  number = true,                                       -- Set numbered lines.
+  relativenumber = true,                               -- Set relative numbered lines.
+  numberwidth = 4,                                     -- set number line column width up to 4(default 4).
+  signcolumn = "yes",                                  -- Always show the sign column, or it would shift the text each time.
+-- Functionality:
+  mouse = "a",                                         -- Allow the mouse to be used in neovim.
+  backup = true,                                       -- Creates a backup file, before edits are written.
+  clipboard = "unnamedplus",                           -- Allows neovim to access the system clipboard.
+  swapfile = true,                                     -- Creates a swapfile.
+  timeoutlen = 1000,                                   -- Time to wait, for a mapped sequence to complete(in milliseconds).
+  undofile = true,                                     -- Enable persistent undo.
+  undodir = "~/.config/nvim/persistant_undo_history",  -- Sets the directory to store persistant-undo saves.
+  updatetime = 300,                                    -- Faster completion(4000ms default).
+  writebackup = true,                                  -- Write a backup file, before saving.
+  completeopt = { "menuone", "noselect" },             -- Mostly just for cmp. ???
+-- Text-formatting:
+  conceallevel = 0,                                    -- Added so `` is visible in markdown files.
+  smartindent = true,                                  -- Keep indention from line-to-line.
+  expandtab = true,                                    -- Convert tabs into spaces.
+  shiftwidth = 4,                                      -- The number of spaces inserted for each indentation
+  tabstop = 4,                                         -- Insert 2 spaces for each tab.
+  wrap = true,                                         -- Wrap my code-lines on the visible screen.
+  guifont = "monospace:h17",                           -- The font used in graphical neovim applications.
+-- Search:
+  hlsearch = true,                                     -- Highlight all matches on when searching.
+  incsearch = true,                                    -- Turn on incremental search highlighting.
+  ignorecase = true,                                   -- Ignore case in search patterns.
+  smartcase = false,                                   -- Overides ignore case, w/ capitals. Default = false.
+  hidden = true,                                       -- Saves all buffers before an exit. if hidden biffers exist.
 }
+-- A lua for-loop to loop through the above table of options.
+-- A table is a list, that is allowed key, value pairs.
+  for k, v in pairs(options) do
+    vim.opt[k] = v
+  end
 
-vim.opt.shortmess:append "c"
+-- Enabling editor pips/EOL/trails/ext/pre...
```

Comment:
I still remember how happy this change always makes in my vim/neo-vim's.
I got this originally from the vi series from the Frugal-computer-guy on you-tube.